src/container/srv_target.c

```diff
@@ -976,17 +991,24 @@ ds_cont_hdl_hash_destroy(struct d_hash_table *hash)
 }
 
 static int
-cont_hdl_add(struct d_hash_table *hash, struct ds_cont_hdl *hdl)
+cont_hdl_add(struct d_hash_table *hash, struct ds_cont_hdl *hdl, struct ds_cont_child *cont)
```

Comment:
minor: the additional 'cont' parameter is unnecessary. the ds_cont_hdl referenced ds_cont_child.